AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_PhysicalSkyAdded.py

```diff
@@ -130,42 +192,188 @@ def AtomEditorComponents_PhysicalSky_AddedToEntity():
-            AtomComponentProperties.physical_sky('Sky Intensity'))
-        Report.result(Tests.sky_intensity, current_sky_intensity == 2)
+            AtomComponentProperties.physical_sky('Intensity Mode'), PHYSICAL_SKY_INTENSITY_MODE["Nit"])
+        Report.result(
+            Tests.intensity_mode_set_to_nit,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Intensity Mode')) == PHYSICAL_SKY_INTENSITY_MODE["Nit"])
+    
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Intensity Mode'), PHYSICAL_SKY_INTENSITY_MODE["Ev100"])
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.intensity_mode_set_to_ev100,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Intensity Mode')) == PHYSICAL_SKY_INTENSITY_MODE["Ev100"])
+
+        # 6. Set Sky Intensity value to 0 then back to 4.
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Sky Intensity'), 0)
+        Report.result(
+            Tests.sky_intensity_set_to_0,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Sky Intensity')) == 0)
+
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Sky Intensity'), 4.0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.sky_intensity_set_to_4,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Sky Intensity')) == 4.0)
+
+        # 7. Set Sun Intensity value to 0 then back to 8.
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Sun Intensity'), 0)
+        Report.result(
+            Tests.sun_intensity_set_to_0,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Sun Intensity')) == 0)
+
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Sun Intensity'), 8.0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.sun_intensity_set_to_8,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Sun Intensity')) == 8.0)
+
+        # 8. Set Turbidity value to 5 then back to 1.
+        physical_sky_component.set_component_property_value(
+            AtomComponentProperties.physical_sky('Turbidity'), 0.0)
+        Report.result(
+            Tests.turbidity_set_to_5,
+            physical_sky_component.get_component_property_value(
+                AtomComponentProperties.physical_sky('Turbidity')) == 0.0)
```

Comment:
Turbidity is an integer so you should be setting and checking for 0 not 0.0

Reply:
fixed